pkg/scanners/scannerv4/scannerv4.go

```diff
@@ -113,9 +114,14 @@ func (s *scannerv4) GetSBOM(image *storage.Image) ([]byte, bool, error) {
 	if err != nil {
 		return nil, false, err
 	}
+
+	imgName := image.GetName()
+	// Desired URI for images: https://<registry>/<repo>-<uuid>
+	uri := fmt.Sprintf("https://%s/%s-%s", imgName.GetRegistry(), imgName.GetRemote(), uuid.NewV4().String())
```

Comment:
we don't want to include the tag nor digest?

Reply:
We proposed that, was a brief discussion, and ultimately the tagless/digestless version was preferred (I just added you to a slack thread with the discussion).

The SPDX document will have the digest and the tag (if referenced by tag) in a different fields FWIW (the name and comment fields)